packages/website/src/pages/layout/ModalWizard.tsx

```diff
@@ -0,0 +1,16 @@
+import {ModalWizardMetadata} from '@coveord/plasma-components-props-analyzer';
+import ModalWizardDemo from '@examples/layout/ModalWizard/ModalWizard.demo';
+
+import {PageLayout} from '../../building-blocs/PageLayout';
+
+export default () => (
+    <PageLayout
+        section="Layout"
+        title="Modal"
+        sourcePath="/packages/mantine/src/components/modal-wizard/ModalWizard.tsx"
+        description="A Modal Wizard is a collection of Modals that user can interact with to display any information step by step"
+        id="ModalWizard"
+        propsMetadata={ModalWizardMetadata}
```

Comment:
These are the props of the old ModalWizard component, you will need to go in the ComponentsList file and generate the props for your new ModalWizard component.